kbase-extension/static/kbase/js/widgets/common/RenderIn.js

```diff
@@ -0,0 +1,13 @@
+define(['jquery'], ($) => {
```

Comment:
I noted in #2378 also, but this seems to be some module duplication / factorization with what's in the `common/errorView` (or something like that) module.

Is there a plan to unify those?

Reply:
Generally, yes. In this case, I simply wanted some convenience functions for working with jQuery.